### DISCUSSION TITLE
Building the table locally?

### DISCUSSION THREAD

**Discussion by iamericfletcher** [osmlab/name-suggestion-index#5978] [General]
Hi, everyone. 

I'm working on adding to `college.json` and am wondering how to rebuild the table so my additions are shown via `http://localhost:63342/name-suggestion-index/docs/index.html?t=brands&k=amenity&v=college`. I've added the following in response to issue #5970:

>  {
      "displayName": "Richard J. Daley College",
      "id": "richardjdaleycollege-62788d",
      "locationSet": {
        "include": ["us-il.geojson"]
      },
      "tags": {
        "amenity": "college",
        "brand": "Richard J. Daley College",
        "brand:wikidata": "Q7326714",
        "brand:wikipedia": "en:Richard J. Daley College",
        "name": "Richard J. Daley College"
      }
    } 

to `college.json`. I then ran `npm run build`, `npm run wikidata`, then opened `docs -> index.html` in the browser. Unfortunately, the table does not include the above addition like I had hoped it would.

I'll work on adding the rest of the colleges once I get this squared away.

**Comment by bhousel** [comment]
Sorry @iamericfletcher , but we don't add individual colleges or universities to this index - we only add the kind of for-profit "universities” that operate like a branded business in multiple locations, such as DeVry or University of Phoenix. I know this is an unusual concept outside of the USA - the “universities” in the name-suggestion-index are really more like training centers than true universities.

This index is really for tracking the types of things where there are a lot of them, not just single things. We are considering adding some university operated assets in #4779

**Reply by iamericfletcher** [reply]
Thank you for the help, @bhousel. I will not continue with the issue, but would like to know if I should or shouldn't expect the table to rebuilt when running the project locally on my machine. I ask since the above addition was not populated in the table after adding it then rebuilding the project according to the documentation.

Thanks!

**Reply by bhousel** [reply]
This project is primarily a data project.  The nsi.guide ReactJS site table will just show whatever data is fetched from the latest published release.  We publish releases about once a week or so.

So, you don't actually have to build the ReactJS project, or run it locally, or expect to see anything change there - I almost never touch that part of the project.
